DataFormats/HcalDigi/bin/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<architecture name="slc6_amd64">
+    <flags LDFLAGS="-Wl,-dynamic-linker,$(GLIBC_BASE)/lib64/ld.so"/>
+</architecture>
+
+<bin name="test_hcal_detid_cuda" file="test_hcal_detid_cuda.cu">
```

Comment:
If this is just a test (as it looks to me) and not a program that should be found from `$PATH`, it should be placed under `test` directory. It should be sufficient to just move both `BuildFile.xml` and `.cu` there.

Reply:
k